sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -361,6 +428,52 @@ def _is_statsbeat_initializing_state(self):
 
     def _is_stats_exporter(self):
         return self.__class__.__name__ == "_StatsBeatExporter"
+    
+    def _should_collect_customer_statsbeat(self):
+        """Check if customer statsbeat collection is enabled.
+        
+        Returns:
+            bool: True if customer statsbeat collection is enabled, False otherwise
+        """
+        # Check environment variable
+        env_value = os.environ.get("APPLICATIONINSIGHTS_STATSBEAT_ENABLED_PREVIEW", "")
+        is_enabled = env_value.lower() in ("true", "1", "yes")
+        # Don't collect customer statsbeat for stats exporter itself or for instrumentation collection
+        return (
+            is_enabled
+            and not self._is_stats_exporter()
+            and not self._instrumentation_collection
+        )
+
+    def _get_telemetry_type(self, envelope: TelemetryItem) -> TelemetryType:
```

Comment:
Same comments apply from the above implementation of `get_telemetry_type` in `storage.py`. Also, if these functions do the same thing, you should abstract it out to `util` so you don't have to define it multiple times.